symbols/overlay29.yml

```diff
@@ -1634,6 +1687,16 @@ overlay29:
         r0: entity pointer
         r1: IQ skill ID
         return: bool
+    - name: UpdateIqSkills
```

Comment:
Was the function signature omitted on purpose for this one? If the params aren't known that's fine, just want to make sure.

Reply:
Oh, looks like I forgot to fill that in. Shouldn't `symbol_check` catch that I didn't add this function to the headers? Because when I run it, it prints nothing.

Reply:
`symbol_check` only enforces things in one direction (everything in the headers must be in the symbols), since it's fine to have symbols that you don't know the exact type for.

I added `symbol_check --extra` a while back to warn you about things in the other direction, which you can run manually, but it's a little noisy. It doesn't make sense to put it in the automation though.